client/src/components/AuthHeader/useStyles.ts

```diff
@@ -7,22 +7,22 @@ const useStyles = makeStyles((theme: Theme) => ({
     flexWrap: 'wrap',
   },
   accAside: {
-    fontSize: 14,
-    color: '#b0b0b0',
-    fontWeight: 400,
-    textAlign: 'center',
-    whiteSpace: 'nowrap',
-    display: 'flex',
-    alignItems: 'center',
-    padding: '1rem',
+    '&&': {
+      fontSize: 14,
+      fontWeight: 900,
+      fontFamily: 'Roboto',
+      textAlign: 'center',
+      whiteSpace: 'nowrap',
+      display: 'flex',
+      alignItems: 'center',
+      padding: '1rem',
+    },
   },
   accBtn: {
-    width: 170,
-    height: 54,
     borderRadius: theme.shape.borderRadius,
     filter: 'drop-shadow(0px 2px 6px rgba(74,106,149,0.2))',
     backgroundColor: '#ffffff',
-    color: '#3a8dff',
+    color: 'red',
```

Comment:
Same as comment above (you can add colors to the palette in the MUI theme and use the theme them)